### ISSUE TITLE
[6.0.0-alpha.20220208] error with loading plugins

### ISSUE THREAD

**Issue by Georg-Git** [tinymce/tinymce#7608]
Init with more than one plugin listed will fail.

e.g. 
plugins: ["table"] is loading the plugin, but
plugins: ["link table"] => no plugin is loaded, 404 errors in Browser Debugger.

https://fiddle.tiny.cloud/YRhaab/5

**Comment by lnewson** [comment]
This is expected, as it's a mixed format of space separated strings and an array of strings. While this did work previously,  we've decided to remove support for it in TinyMCE 6 as it was undocumented and is rather confusing/misleading (see https://github.com/tinymce/tinymce/pull/7601). If you update the fiddle to use the [documented](https://www.tiny.cloud/docs/configure/integration-and-setup/#plugins) version (e.g. a space separated string) you'll see it loads the plugins correctly: https://fiddle.tiny.cloud/YRhaab/6

**Comment by Georg-Git** [comment]
Thanks!

So - You also have to update the default example in fiddle:
https://fiddle.tiny.cloud/baaaab

With regard to the renaming of configurations - two different default examples, one v5 and one v6?

**Comment by lnewson** [comment]
Yeah, the default fiddle will be getting updated either today or tomorrow I believe to use something similar to the example I gave (I'm not 100% sure on where it's at sorry).
